packages/itwinui-css/src/anchor/anchor.scss

```diff
@@ -20,7 +20,7 @@
     @include iui-anchor-underline('on-initial');
   }
 
-  [data-iui-contrast='high'] & {
+  :is([data-iui-contrast='high'] &) {
```

Comment:
without this change, the browser will look for `[data-iui-contrast=high]` inside `:where(.iui-root, [data-iui-theme])` (which is applied through postcss). this doesn't work because both of those are applied on the same element.

with this change, the browser will first resolve `[data-iui-contrast=high] .iui-anchor`, and then separately make sure that `iui-anchor` is also inside `:where(.iui-root, [data-iui-theme])`.

for more details, see https://www.bram.us/2023/01/17/using-is-in-complex-selectors-selects-more-than-you-might-initially-think/